WalletWasabi.Gui/Assets/LegalIssues.txt

```diff
@@ -1,7 +1,7 @@
 
 Legal statement of Wasabi Wallet
 
-The purpose of Wasabi Wallet is to provide privacy for its users’ Bitcoin transactions. This means that with the use of the wallet we would like to enable the users to make Bitcoin payments in a way that their spending history, address balances, etc. are not visible for the receiver party. This does not necessarily mean that the user stays anonymous, since the receiver party might have to identify the sending party.
+The purpose of Wasabi Wallet is to provide privacy for its users’ Bitcoin transactions. This means that with the use of the Wallet we would like to enable the users to make Bitcoin payments in a way that their spending history, address balances, etc. are not visible for the receiver party. This does not necessarily mean that the user stays anonymous, since the receiver party might have to identify the sending party.
```

Comment:
```suggestion
The purpose of Wasabi Wallet is to provide privacy for its users’ Bitcoin transactions. This means that with the use of the wallet we would like to enable the users to make Bitcoin payments in a way that their spending history, address balances, etc. are not visible for the receiver party. This does not necessarily mean that the user stays anonymous, since the receiver party might have to identify the sending party.
```

ACK for everything else.